### PR TITLE
Update README, AboutUs, UG and DG.

### PR DESCRIPTION
**Pull request by gohyongjing**
### Changes:
* Update product description in `README.md`
* Add personal image links and roles in `AboutUs.md`
* Add Target User profile in `DeveloperGuide.md`
* Add User Stories related to pairs in `DeveloperGuide.md`
* Add description and examples for `add_pair` and `delete_pair` in `UserGuide.md`
* Add personal Project Porfolio Page skeleton `gohyongjing.md`

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -72,28 +88,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Pairs Volunteer and Elderly**
```
Comment (zhacatomn) [inline docs/DeveloperGuide.md:91]: For the use cases, isn't @heeeyi suppose to fill it in? (At least according to the minutes). I'm fine either way.

**Review by zhacatomn — APPROVED** [review]
Yup LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (heeeyi) [reply docs/DeveloperGuide.md:91]: I have done this part for all the Use cases :)

File: README.md [hunk 2]
```diff
@@ -4,13 +4,9 @@
...
+FriendlyLink streamlines volunteer and elderly management for single administrators of small NPOs. With its easy-to-use text-based interface and contact management features, say goodbye to manual record-keeping and hello to a more efficient and organised way of managing the volunteers’ and elderly’s contact details.
+
+* The project is an ongoing software project for a desktop application (called _FriendlyLink_) used for managing details of elderly members and volunteers for Non-Profit Organisations.
+* It is **written in OOP fashion** and comes with a **reasonable level of user and developer documentation**.
+* For the detailed documentation of this project, see the **[FriendlyLink Product Website]()**.
+* This project is modified from [AddressBook-Level3 project](https://se-education.org/addressbook-level3) created by the [SE-EDU initiative](https://se-education.org). If you would like to contribute code to the original project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
```
Comment (heeeyi) [inline README.md:12]: This is nice writing!

**Review by heeeyi — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (gohyongjing) [reply docs/DeveloperGuide.md:91]: Alright, nice!

**Comment by gohyongjing** [comment]
Thanks everyone! Will merge the PR now

**Pull request merged**